doc/base.inc

```diff
@@ -488,18 +517,32 @@ INTERNAL_DOCS          = NO
 CASE_SENSE_NAMES       = NO
 
 # If the HIDE_SCOPE_NAMES tag is set to NO then doxygen will show members with
-# their full class and namespace scopes in the documentation. If set to YES the
+# their full class and namespace scopes in the documentation. If set to YES, the
 # scope will be hidden.
 # The default value is: NO.
 
 HIDE_SCOPE_NAMES       = NO
 
+# If the HIDE_COMPOUND_REFERENCE tag is set to NO (default) then doxygen will
+# append additional text to a page's title, such as Class Reference. If set to
+# YES the compound reference will be hidden.
+# The default value is: NO.
+
+HIDE_COMPOUND_REFERENCE= NO
```

Comment:
I propose setting this to `YES`, so that Doxygen's output better matches what we get from Sphinx (e.g., https://pipelines.lsst.io/v/DM-11253/py-api/lsst.pipe.base.Task.html). I've confirmed locally that this gives "<namespace::class>" rather than "<namespace::class> Class Reference".